resources/js/components/LiveEvents.vue

```diff
@@ -123,10 +122,34 @@ export default {
 	data ()
     {
 		return {
-			events: []
+			events: [],
+            clicks: 0,
+            timer: null
 		};
 	},
 	methods: {
+        /**
+         * This is usually how double-clicks are handled
+         * without overlapping with the click events
+         * @see https://stackoverflow.com/a/41309853/5828796
+         * @param event
+         * @param index
+         */
+        click(event, index)
```

Comment:
Vue also has a `@dbClick` handler

Reply:
That's what we have currently on production, and that works. This issue is, however, that it does not prevent the 'click' event from being fired as well. All solutions were like this.